src/Plugin/Condition/NodeHasTerm.php

```diff
@@ -144,14 +198,13 @@ public function submitConfigurationForm(array &$form, FormStateInterface $form_s
           $uris[] = $uri;
         }
       }
-      if (!empty($uris)) {
-        $this->configuration['uri'] = implode(',', $uris);
-      }
     }
 
-    $this->configuration['logic'] = $form_state->getValue('logic');
-
-    parent::submitConfigurationForm($form, $form_state);
```

Comment:
this `parent::submitConfigurationForm($form, $form_state);` call needs to stay or else checking the `Negate the condition` checkbox does not save. Evidenced by a site with this PR's patch applied: https://d.pr/i/Ju86gL

Reply:
The submit was just moved to the top of the function.

Reply:
Also, this presently isn't usable for block placement. The thing I'm pretty sure you're missing there is the selection of something to satisfy the `node` context, which is required?

Reply:
Should now be addressed, I think.